crates/wit-bindgen/src/lib.rs

```diff
@@ -2412,29 +2427,35 @@ impl<'a> InterfaceGenerator<'a> {
             } else {
                 uwrite!(self.src, "Ok(r)\n");
             }
-        } else if let Some((_, err, _)) = self.special_case_trappable_error(&func.results) {
-            let err = &self.resolve.types[resolve_type_definition_id(self.resolve, err)];
-            let err_name = err.name.as_ref().unwrap();
-            let owner = match err.owner {
-                TypeOwner::Interface(i) => i,
-                _ => unimplemented!(),
-            };
-            let convert_trait = match self.path_to_interface(owner) {
-                Some(path) => format!("{path}::Host"),
-                None => format!("Host"),
-            };
-            let convert = format!("{}::convert_{}", convert_trait, err_name.to_snake_case());
-            uwrite!(
-                self.src,
-                "Ok((match r {{
+        } else {
+            self.gen
+                .used_trappable_imports_opts
+                .insert(func.name.clone());
```

Comment:
Could this be added to `special_case_trappable_error`? (perhaps by updating that to take `&Function`)?